sql/mob_pools.sql

```diff
@@ -3177,7 +3177,7 @@ INSERT INTO `mob_pools` VALUES (3121,'Petrocrab','Petrocrab',77,0x00006401000000
 INSERT INTO `mob_pools` VALUES (3122,'Petro_Pukis','Petro_Pukis',87,0x0000A40100000000000000000000000000000000,1,1,12,240,100,0,1,1,1,2,513,0,1718,135,0,0,0,0,0,87,87);
 INSERT INTO `mob_pools` VALUES (3123,'Pevv_the_Riverleaper','Pevv_the_Riverleaper',213,0x00001F0500000000000000000000000000000000,14,14,8,240,100,0,1,0,1,18,0,0,0,0,0,0,0,0,0,712,213);
 INSERT INTO `mob_pools` VALUES (3124,'Pey','Pey',227,0x00003C0200000000000000000000000000000000,1,1,5,240,100,0,1,0,1,8,0,0,2607,133,0,0,0,0,0,227,227);
-INSERT INTO `mob_pools` VALUES (3125,'Phanduron_the_Condemned','Phanduron_the_Condemned',221,0x00001A0200000000000000000000000000000000,4,1,3,360,100,0,1,0,1,2,0,32,0,157,0,0,2,0,0,221,221);
+INSERT INTO `mob_pools` VALUES (3125,'Phanduron_the_Condemned','Phanduron_the_Condemned',221,0x00001A0200000000000000000000000000000000,4,1,3,360,100,0,1,0,1,2,24,32,0,157,0,0,2,0,0,221,221);
```

Comment:
mobtype 24 would be battlefield + called. Is this some kind of mini instance where outside players aren't allowed to cast on fighting players?